legend-engine-xt-serviceStore-executionPlan/src/test/resources/securitySchemes/testGrammar.pure

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+###Pure
+Class meta::external::store::service::showcase::domain::S_Trade
+{
+  s_tradeId       : String[1];
+  s_traderDetails : String[1];
+  s_tradeDetails  : String[1];
+}
+
+###Mapping
+Mapping meta::external::store::service::showcase::mapping::ServiceStoreMapping
+(
+  *meta::external::store::service::showcase::domain::S_Trade[s_trade_set]: ServiceStore
+  {
+     ~service [meta::external::store::service::showcase::store::TradeProductServiceStore] TradeServices.AllTradeService
+  }
+)
+
+###ExternalFormat
+SchemaSet meta::external::store::service::showcase::store::tradeSchemaSet
+{
+  format  : FlatData;
+  schemas : [
+    {
+        content: 'section A: DelimitedWithHeadings{  scope.untilEof;  delimiter: \',\';  Record  {      s_tradeId       : STRING;      s_traderDetails : STRING;      s_tradeDetails  : STRING;  }}';
+    }
+  ];
+}
+
+Binding meta::external::store::service::showcase::store::tradeServiceStoreSchemaBinding
+{
+  schemaSet     : meta::external::store::service::showcase::store::tradeSchemaSet;
+  contentType   : 'application/x.flatdata';
+  modelIncludes : [ meta::external::store::service::showcase::domain::S_Trade ];
+}
+
+###Runtime
+Runtime meta::external::store::service::showcase::runtime::ServiceStoreRuntime
+{
+    mappings :
+    [
+        meta::external::store::service::showcase::mapping::ServiceStoreMapping
+    ];
+    connections :
+    [
+		meta::external::store::service::showcase::store::TradeProductServiceStore :
```

Comment:
indentation

Reply:
this looks okay in ide. Not sure why its like this here